orbs/shared/commands/setup_devenv.yaml

```diff
@@ -12,9 +12,13 @@ parameters:
     description: Arguments for provision, optional
     type: string
     default: ""
+  devenv_pre_release:
+    description: Provision the devenv using pre-release version
+    type: boolean
+    default: false
```

Comment:
This isn't actually needed, because we're not passing and argument to the step. We just need the environment variable addition in the e2e job.

However, this e2e job is only used in the new stencil module. You'll want to modify the file in `pkg/codegen/templates/.circleci/config.yml` for everything else currently.